### PR TITLE
Block translation

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by core23** [comment]
@Soullivaneuh Can you help me with the French translation?

File: Resources/translations/SonataNewsBundle.en.xliff [hunk 1]
```diff
@@ -427,6 +427,14 @@ Quick moderation links :
                 <source>group_classification</source>
                 <target>Classification</target>
             </trans-unit>
+            <trans-unit id="sonata_news_block_recent_comments">
+                <source>sonata.news.block.recent_comments</source>
+                <target>News - Letzte Kommentare</target>
+            </trans-unit>
+            <trans-unit id="sonata_news_block_recent_posts">
+                <source>sonata.news.block.recent_posts</source>
+                <target>News - Letzte Artikel</target>
```
Comment (OskarStark) [inline Resources/translations/SonataNewsBundle.en.xliff]: wrong

File: Resources/translations/SonataNewsBundle.de.xliff [hunk 2]
```diff
@@ -435,6 +435,14 @@
                 <source>group_classification</source>
                 <target>Klassifizierung</target>
             </trans-unit>
+            <trans-unit id="sonata_news_block_recent_comments">
+                <source>sonata.news.block.recent_comments</source>
+                <target>News - Recent comments</target>
+            </trans-unit>
+            <trans-unit id="sonata_news_block_recent_posts">
+                <source>sonata.news.block.recent_posts</source>
+                <target>News - Recent posts</target>
```
Comment (OskarStark) [inline Resources/translations/SonataNewsBundle.de.xliff]: wrong, you exchanged the de and en file :smile:

File: Resources/translations/SonataNewsBundle.fr.xliff [hunk 3]
```diff
@@ -427,6 +427,14 @@ Liens de modération rapide :
                 <source>group_classification</source>
                 <target>Classification</target>
             </trans-unit>
+            <trans-unit id="sonata_news_block_recent_comments">
```
Comment (OskarStark) [inline Resources/translations/SonataNewsBundle.fr.xliff]: btw. why is the id different to the source key?

i think in all other files we keep them the same, don't we?

Comment (core23) [inline Resources/translations/SonataNewsBundle.fr.xliff]: Don't know the exact reason, but all keys have replaced the dots with underscores.

File: Resources/translations/SonataNewsBundle.de.xliff (hunk 2 above)
Comment (core23) [inline Resources/translations/SonataNewsBundle.de.xliff]: Stupid copy paste issue. Thanks :+1:

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**